src/orbit/lib/arrays.js

```diff
@@ -24,4 +24,16 @@ function some(array, predicate) {
   return false;
 }
 
-export { every, some };
+function firstResult(array, valueFn) {
```

Comment:
Is this not https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find ?